### PR TITLE
Added the BMI User Acceptance Testing Framework

### PR DESCRIPTION
**Pull request by pgrosu**
Added the BMI User Acceptance Testing Framework

### CODE REVIEW

File: tests/User Acceptance Testing Framework/scripts/setup_acceptance_test_scripts.cfg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+pr-60:source_initiate-acceptance-tests.sh-pr-60
```
Comment (naved001) [inline tests/User Acceptance Testing Framework/scripts/setup_acceptance_test_scripts.cfg]: we already merged pr-60, what's this for?

File: tests/User Acceptance Testing Framework/scripts/setup_acceptance_test_environment/source_initiate-acceptance-tests.sh-pr-60 [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+if [ ! -z  "`sudo ls /etc/ | grep redhat-release`" ]; then
+  cp /etc/hostname .
+  cat /etc/hostname | cut -f1 -d'.' > hostname
+  sudo cp hostname /etc
+fi
+
+git clone https://github.com/CCI-MOC/ims.git
+cd ims
+git fetch origin pull/60/head:pr-60
+git checkout pr-60
```
Comment (naved001) [inline tests/User Acceptance Testing Framework/scripts/setup_acceptance_test_environment/source_initiate-acceptance-tests.sh-pr-60]: okay, so this file is for testing out pr-60 which we have already merged. Does this have any other purpose? Can you put that in the file description on top.

File: tests/User Acceptance Testing Framework/scripts/setup-bmi-book-chapter-2-environment.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+git clone https://github.com/CCI-MOC/ims.git
+cd ims
+git fetch origin pull/60/head:pr-60
+git checkout pr-60
```
Comment (naved001) [inline tests/User Acceptance Testing Framework/scripts/setup-bmi-book-chapter-2-environment.sh]: stuff related to pr-60

File: tests/User Acceptance Testing Framework/scripts/rollback.py [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+    
+    bmi_snapshot_node_stdout = check_output('bmi snap rm ' + project_name + ' ' + snapshot_name, stderr=STDOUT, shell=True)
+    bmi_snapshot_node_status = bmi_snapshot_node_stdout.strip()
+    try:
+        # Ensure that the image clone actually exists in BMI and RBD
+        print( "      -> Checking that " + ceph_clone_name + " is removed from BMI's database...")
+        bmi_db_ls_stdout = check_output('bmi db ls | grep ' + ceph_clone_name, stderr=STDOUT, shell=True)
+    except CalledProcessError:
+        pass
+        assert True
```
Comment (naved001) [inline tests/User Acceptance Testing Framework/scripts/rollback.py]: what if you just pass and not assert True?

**Comment by chemistry-sourabh** [comment]
@pgrosu please remove the unnecessary commits from this PR

**Comment by chemistry-sourabh** [comment]
The second question I have is should these tests be run in the CI ? If yes I feel they will need to simplified and should fit in with respect of the tests.

**Comment by naved001** [comment]
@pgrosu do you think you would have time anytime soon to address the comments on your PRs

**Pull request closed without merging**